### PR TITLE
Omission of !important from changelocal.component.css

### PR DESCRIPTION
**Pull request by harshit151001**
`.labelselect label {`
`  color: $labeldropdown !important;`
`}` 
to
`.labelselect label {`
`  color: $labeldropdown;`
`}` 

File changed : src/components/choose-locale/changelocal.component.css

### CODE REVIEW

**Comment by HerbertYiga** [comment]
@harshit151001  can you first resolve the merge conflicts on this

**Comment by harshit151001** [comment]
Done @HerbertYiga

**Comment by HerbertYiga** [comment]
> Done @HerbertYiga

Add a description for your changes too

**Comment by harshit151001** [comment]
> > Done @HerbertYiga
> 
> Add a description for your changes too

```
.labelselect label {
color: $labeldropdown !important;
}
```
to
```
.labelselect label {
color: $labeldropdown;
}
```
File changed : src/components/choose-locale/changelocal.component.css

File: src/components/choose-locale/changelocal.component.scss [hunk 1]
```diff
@@ -1,5 +1,7 @@
 @import "../../root.scss";
 
 .labelselect label {
-  color: $labeldropdown !important;
```
Comment (HerbertYiga) [inline src/components/choose-locale/changelocal.component.scss:4]: @harshit151001  thanks cc @FlorianRappl

**Review by HerbertYiga — APPROVED** [review]

**Review by denniskigen — APPROVED** [review]
Nice work @harshit151001

**Comment by harshit151001** [comment]
should i delete this branch ?

**Comment by denniskigen** [comment]
@harshit151001 yep

**Pull request merged**